src/main/resources/META-INF/primefaces-p.taglib.xml

```diff
@@ -11202,6 +11202,14 @@
             <required>false</required>
             <type>java.lang.Boolean</type>
         </attribute>
+        <attribute>
+            <description>
+                <![CDATA[Binary data to stream the image dynamically]]>
+            </description>
+            <name>stream</name>
+            <required>false</required>
+            <type>java.lang.Object</type>
+        </attribute>
```

Comment:
This needs to be removed right? and `image` needs to be updated to java.lang.Object.

Reply:
Done! In addition, `image` is required='true', confirm?